src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -60,6 +63,14 @@ private[query] class EventsByTagPublisher(
     case None       => Long.MaxValue
   }
 
+  // Subscribe to DistributedPubSub so we can receive immediate notifications when the journal has written something.
+  if (eventualConsistencyDelayMillis == 0) {
```

Comment:
Sure. I originally didn't want to depend on configs from `CassandraJournal`, because one might have a cluster setup of "only query side" nodes with no write journal running on them. It's just a semantical distinction though; I'll use `pubsub-minimum-interval` as the switch.